CHANGELOG.md

```diff
@@ -49,6 +50,7 @@
   * BraintreeCore
     * Remove `BTConfiguration+Extensions.isUnionPayEnabled` property
     * Remove `BTPreferredPaymentMethods` and `BTPreferredPaymentMethodResult`
+    * Remove `BTKeychain` class and methods
```

Comment:
Nit - This wasn't public, was it? Don't think it needs an entry.

Reply:
It was public before ([here in v6](https://github.com/braintree/braintree_ios/blob/master/Sources/BraintreeCore/BTKeychain.swift) but [wasn't in v5](https://github.com/braintree/braintree_ios/blob/5.x/Sources/BraintreeCore/BTKeychain.h)). Genuinely no idea why we made it public unless it was for some weird interop reason during the conversion? But even then 🤷 . Worth calling it out since it technically a breaking change between betas or just remove this line?